<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tablesDataV1</class>
 <widget class="QDialog" name="tablesDataV1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BD JO</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>7</number>
     </property>
     <widget class="QWidget" name="tabEpreuves">
      <attribute name="title">
       <string>LesEpreuves</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableEpreuves">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>7</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>110</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>numEp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>nomEp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>formeEp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>nomDi</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>categorieEp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>nbSportifsEp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>dateEp</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_epreuves">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>120</red>
               <green>120</green>
               <blue>120</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLesEquipiers">
      <attribute name="title">
       <string>LesEquipes</string>
      </attribute>
      <widget class="QTableWidget" name="tableLesEquipiers">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>926</width>
         <height>488</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>110</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>numEq</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>numSp</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_LesEquipiers">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>510</y>
         <width>926</width>
         <height>13</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabSportifs">
      <attribute name="title">
       <string>LesSportifs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTableWidget" name="tableSportifs">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>110</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>numSp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>nomSp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>prenomSp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>pays</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>categorieSp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>dateNaisSp</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_sportifs">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>120</red>
               <green>120</green>
               <blue>120</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabinscriptions">
      <attribute name="title">
       <string>LesInscriptions</string>
      </attribute>
      <widget class="QTableWidget" name="tableinscriptions">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>926</width>
         <height>488</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>110</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>numIn</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>numEp</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_inscriptions">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>500</y>
         <width>926</width>
         <height>13</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabresultats">
      <attribute name="title">
       <string>LesResultats</string>
      </attribute>
      <widget class="QTableWidget" name="tableiresultats">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>926</width>
         <height>488</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>110</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>numEp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>numEp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>silver</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>bronze</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_resultats">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>500</y>
         <width>926</width>
         <height>13</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLesAgesSportifs">
      <attribute name="title">
       <string>LesAgesSportifs</string>
      </attribute>
      <widget class="QTableWidget" name="tableLesAgesSportifs">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>926</width>
         <height>488</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>110</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>numEp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>nomSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>prenomSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>pays</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>categorieSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>dateNaisSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ageSp</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_LesAgesSportifs">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>500</y>
         <width>926</width>
         <height>13</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLesNbsEquipiers">
      <attribute name="title">
       <string>LesNbsEquipiers</string>
      </attribute>
      <widget class="QTableWidget" name="tableLesNbsEquipiers">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>926</width>
         <height>488</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>110</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>numEq</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>nbEquipiersEq</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_LesNbsEquipiers">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>500</y>
         <width>926</width>
         <height>13</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tablogs">
      <attribute name="title">
       <string>logs</string>
      </attribute>
      <widget class="QTableWidget" name="tablelogs">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>926</width>
         <height>488</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>110</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>numEp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>datemaj</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_logs">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>500</y>
         <width>926</width>
         <height>13</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_data_V1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_data_V1">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2_data_V1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_data_V1</sender>
   <signal>clicked()</signal>
   <receiver>tablesDataV1</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshSpectaclesData()</slot>
  <slot>refreshZonesData()</slot>
  <slot>refreshRepresentationsData()</slot>
 </slots>
</ui>
